src/Client/Lookup.js

```diff
@@ -80,7 +80,12 @@ module.exports = class Lookup {
     const parallel = await Promise.all(requests);
     let accounts = [];
     Object.keys(parallel).forEach((result) => accounts.push(parallel[result]));
-    accounts = accounts.flat(1);
+
+    if (Array.prototype.flat) {
+      accounts = accounts.flat(1);
+    } else {
+      accounts = Utils.flat(accounts);
```

Comment:
Just a comment regarding this,
This is not a real issue, since the library is set to be used Node v11 or above which includes this functionality to begin with. TLS is v12 currently.